### PR TITLE
Update engine version to release-20200503.

### PR DESCRIPTION
**Pull request by MustaphaTR**

### CODE REVIEW

File: OpenRA.Mods.RA2/Traits/PeriodicExplosion.cs [hunk 1]
```diff
@@ -103,9 +103,17 @@ void ITick.Tick(Actor self)
...
+				var args = new WarheadArgs
+				{
+					Weapon = weapon,
+					DamageModifiers = self.TraitsImplementing<IFirepowerModifier>().Select(a => a.GetFirepowerModifier()).ToArray(),
+					Source = self.CenterPosition,
+					SourceActor = self,
+					WeaponTarget = Target.FromPos(self.CenterPosition + localoffset)
+				};
+
+				weapon.Impact(Target.FromPos(self.CenterPosition + localoffset), args);
```
Comment (abcdefg30) [inline OpenRA.Mods.RA2/Traits/PeriodicExplosion.cs:116]: We can probably re-use `args.WeaponTarget` here instead of calculating it again, can't we?

**Comment by abcdefg30** [comment]
Travis is also reporting errors.

File: mods/ra2/mod.yaml [hunk 2]
```diff
@@ -113,6 +113,7 @@ ChromeLayout:
 	ra2|chrome/assetbrowser.yaml
```
Comment (Mailaender) [inline mods/ra2/mod.yaml:113]: The whole thing can go away now and use the common one.

**Comment by MustaphaTR** [comment]
Updated. Travis just failed to download Eluant for some reason, restarting it fixed the check.

**Comment by Phrohdoh** [comment]
Any idea why `mods/ra2/uibits/musicplayer.png`'s file size increased substantially?

**Comment by abcdefg30** [comment]
> Any idea why `mods/ra2/uibits/musicplayer.png`'s file size increased substantially?

**Comment by Mailaender** [comment]
I don't think this is any real problem, but here you go:

```
pngcrush musicplayer.png 
  Recompressing IDAT chunks in musicplayer.png
   Total length of data found in critical chunks            =       642
pngcrush: iCCP: known incorrect sRGB profile
   Best pngcrush method        =   4 (ws 15 fm 0 zl 9 zs 1) =       630
CPU time decode 0.000701, encode 0.006188, other 0.001267, total 0.008675 sec
```

[musicplayer.png](https://user-images.githubusercontent.com/756669/93019200-dd6d7a00-f5d5-11ea-9d90-b08f62165e4f.png)

**Comment by MustaphaTR** [comment]
Updated.

**Comment by MustaphaTR** [comment]
New playtest is coming, but this is still sitting here. Could somebody review this?

**Comment by Mailaender** [comment]
This suffers from https://github.com/MustaphaTR/Romanovs-Vengeance/issues/46 when using high DPI which can be fixed by removing https://github.com/OpenRA/ra2/blob/bdde553cd57873ff26e30f3e1f8b8c7f139078bc/mods/ra2/chrome/ingame-player.yaml#L490-L499

**Comment by MustaphaTR** [comment]
Updated.

**Comment by Mailaender** [comment]
See also https://github.com/MustaphaTR/ra2/pull/5.

**Comment by MustaphaTR** [comment]
Updated.

**Pull request merged**